### PR TITLE
netstat change uint size from 32 to 64 bits

### PR DESCRIPTION
**Pull request by binjip978**
Signed-off-by: binjip978 <binjip978@gmail.com>

### CODE REVIEW

**Comment by binjip978** [comment]
fixes https://github.com/prometheus/procfs/issues/418 
also minor change to struct layout to use 16 bytes instead of 24 according to fieldalignment analyser.

File: netstat.go [hunk 1]
```diff
@@ -23,8 +23,8 @@ import (
 
 // NetStat contains statistics for all the counters from one file
 type NetStat struct {
-	Filename string
 	Stats    map[string][]uint64
+	Filename string
```
Comment (SuperQ) [inline netstat.go:27]: This change seems unrelated.

Comment (SuperQ) [reply netstat.go:27]: Oh, sorry, I missed the comment on struct alignment. Thanks!

**Review by SuperQ — APPROVED** [review]
LGTM
